src/components/DocumentVideo/CaptureControls/useVideoCaptureStep.ts

```diff
@@ -0,0 +1,114 @@
+import useCaptureStep from '../../utils/useCaptureStep'
+import type { CaptureFlows, CaptureSteps } from '~types/docVideo'
+
+const STEPS_BY_FLOW: Record<CaptureFlows, CaptureSteps[]> = {
+  passport: ['intro', 'front'],
+  cardId: ['intro', 'front', 'back'],
+  paperId: ['intro', 'front', 'back'],
+}
+
+export type MultiFrameCaptureStepActions =
+  | 'showButton'
+  | 'hideButton'
+  | 'holdStill'
+  | 'success'
+  | 'submit'
+
+export type MultiFrameCaptureSteps = 'intro' | 'front' | 'back'
```

Comment:
IMO we should rename these constants to avoid ambiguous `MultiFrameCapture` prefix with `useMultiFrameCaptureStep`

Reply:
Done